### PR TITLE
refactor(Popover): replace react-pose by framer-motion for the animation

### PR DESCRIPTION
**Pull request by spartDev**
- [ ] Feature
- [ ] Fix
- [x] Enhancement

## Description
Replace react-pose by framer-motion for the animation.
This PR also remove react-pose dependecy

## Requirements :

:warning: Requires running `yarn` command.

## Todo - Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: src/Dropdown/__tests__/Dropdown.test.js [hunk 1]
```diff
@@ -20,9 +20,9 @@ describe('<Dropdown />', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
-  test('should toggle the dropdown', done => {
+  test('should toggle the dropdown', async () => {
     const props = { title: 'title' };
-    const { container, queryAllByText } = render(
+    const { container, queryAllByText, debug } = render(
```
Comment (sun-tea) [inline src/Dropdown/__tests__/Dropdown.test.js]: debug

Comment (anglol) [inline src/Dropdown/__tests__/Dropdown.test.js]: Remove debug.

**Comment by sun-tea** [comment]
When using with portal for Dropdown component, the popover is misplaced

**Comment by anglol** [comment]
I just made the correction for the portal. Review?

**Comment by spartDev** [comment]
I can not approve this PR because is mine, @sun-tea @Artikodin ca you check it

**Pull request merged**